woocommerce/pinterest-for-woocommerce
# Add product attributes

### Changes proposed in this Pull Request:

Resolves part of #195 

This PR adds the product attribute functionality based on the Google Listings and Ads extension. This initial PR copies the basic framework for adding product attributes and includes the first attribute (condition).

Overview of changes:
- Add PHP view framework for displaying input elements in the backend
- Add Admin class for loading the backend resources
- Add product attributes and a manager to handle multiple attributes
- Add initial condition attribute
- Compile CSS and load it for product editing pages
- Add the attributes to the XML feed

### Detailed test instructions:

1. Update to this PR 
2. Set the condition in the Pinterest tab when editing several products
3. Set the conditions in the Pinterest tab when editing several variations
4. Regenerate the feed and monitor the list of the issues (Note: I haven't found a good way to speed up the regeneration of a feed besides waiting 24h)
5. Check the issues list and confirm that the condition warning is no longer shown for the products you edited.
![image](https://user-images.githubusercontent.com/11388669/146349469-9ebac170-f3ac-4e2b-b063-eb167dacc81c.png)
6. Check the XML feed and confirm we see the condition added as follows:
```
<g:condition>new</g:condition>
```

### Changelog entry

* Add - Product attribute for condition.
